apps/aecore/src/aec_keys.erl

```diff
@@ -404,7 +404,7 @@ check_env() ->
     Pwd =
         case aeu_env:user_config([<<"keys">>, <<"password">>]) of
             undefined ->
-                {ok, P} = application:get_env(aecore, password),
+                P = application:get_env(aecore, password, <<"default_pwd">>),
```

Comment:
how do we deal with lack of password? Quick read shows that it will fail in hashining (for decription) call. Maybe we should shut down the node here and require password?